packages/libs/components/src/plots/LinePlot.tsx

```diff
@@ -270,41 +296,49 @@ const LinePlot = makePlotlyPlotComponent('LinePlot', (props: LinePlotProps) => {
         // NOTE: unfortunately the newlines do not render.  Newlines can be added to the
         // 'hovertemplate' but from there I don't think we can access arbitrary values,
         // such as 'upper', 'lower' and 'n'
-        .map((series) => ({
-          ...series,
-          hovertemplate: '%{text}',
-          text: zip(
-            series.binLabel ?? [],
-            (series.x ?? []).map(String),
-            (series.y ?? []).map(String),
-            (series.yErrorBarLower ?? []).map(String),
-            (series.yErrorBarUpper ?? []).map(String),
-            series.extraTooltipText ?? []
-          ).map(([binLabel, x, y, lower, upper, xtra], index) => {
-            const binSampleSize = series.binSampleSize?.[index];
-            const yText =
-              binSampleSize &&
-              'denominatorN' in binSampleSize &&
-              binSampleSize.denominatorN === 0
-                ? 'Undefined'
-                : y;
-            const CI =
-              lower != null && upper != null
-                ? ` (95% CI: ${lower} - ${upper})`
-                : '';
-            const N = binSampleSize
-              ? 'N' in binSampleSize
-                ? binSampleSize.N
-                : binSampleSize.denominatorN
-              : undefined;
-            // use <br> instead of \n for line break
-            return (
-              `x: ${binLabel ?? x}<br>y: ${yText}${CI}` +
-              (N !== undefined ? '<br>n: ' + N : '') +
-              (xtra ? '<br>' + xtra : '')
-            );
-          }),
-        })),
+
+        // need to exclude marginal histogram
```

Comment:
a better comment would be

```
// don't process tooltips for marginal histogram
```

But I am confused - where is `series.mode` set to non-null?  Seems like the consumer `LineplotVisualization` where the value is `'lines+markers'`.  This is a bit opaque.  Is there any way we can make it more self-explanatory?

I think at the minimum some very clear comments should be added. It seems like a specification for the props passed to this component is that if `showMarginalHistogram` is true, then a data series containing the histogram counts with undefined `mode`, must be provided.  Perhaps even validate the props to make sure this is the case.